### PR TITLE
Issue 32

### PR DESCRIPTION
**Pull request by KhalilMrini**
Added the title to the json format and added it to the ac-form package. #32

### CODE REVIEW

**Comment by KhalilMrini** [comment]
Transferred changes to ac-quiz

**Comment by houshuang** [comment]
I tried creating a quiz, and then displaying it, but I get an error. Here's the activity data of the quiz I created:

```
{
    "_id" : "98c6ba7b-e82a-432a-9678-4330519470b2",
    "activity_type" : "ac-quiz",
    "data" : {
        "name" : "Hello",
        "MCQ" : [ 
            {
                "title" : "How many boxes are there? ",
                "answers" : [ 
                    {
                        "title" : "Yes",
                        "answer" : true
                    }, 
                    {
                        "title" : "No",
                        "answer" : false
                    }
                ],
                "details" : "Something something"
            }, 
            {
                "title" : "How could that be?",
                "answers" : [ 
                    {
                        "title" : "Peter",
                        "answer" : false
                    }, 
                    {
                        "title" : "Nichol",
                        "answer" : true
                    }
                ]
            }
        ]
    },
    "created_at" : ISODate("2016-11-14T22:21:23.714Z")
}
```

Error:
```
modules.js?hash=eae57a5…:136857 Uncaught TypeError: Cannot read property 'split' of undefined(…)
```

File: ac/ac-quiz/src/index.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            type: "string",
+            title: "Enter an explanation",
+          }
+        }
+      }
+    }
+  }
+}
+
+// Obviously assumes even array
```
Comment (houshuang) [inline ac/ac-quiz/src/index.js]: Remove this line, left over from old code

File: ac/ac-quiz/src/index.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+export const ActivityRunner = ({ config, logger, onCompletion }) => {
+  const propdef = config.questions.split(',').reduce(
```
Comment (houshuang) [inline ac/ac-quiz/src/index.js]: It seems like you only implemented the configuration and not the logic to display the form? This is from a very simple form where the questions are all in one line, separated by commas. Your form needs different logic to display it.

Comment (houshuang) [inline ac/ac-quiz/src/index.js]: Please complete displaying of quizzes as well before we can accept the pull request.

**Comment by lfaucon** [comment]
@houshuang seems like @KhalilMrini was not finished working on building ac-quiz. I agreed with him that he should push anything he wants on his branch even unfinished states of the app, but assign one of us on the pull request when he is ready to have his work reviewed and merged.

**Comment by KhalilMrini** [comment]
Yes @lfaucon is right but next time I will not do a pull request until I finish as it is confusing. Thank you very much @houshuang for the feedback!

**Comment by houshuang** [comment]
Hi Khalil, no problem - sorry, seeing a pull request, I assumed it was ready to be merged. Feel free to ping me when it's done, I look forward to including this code.

File: ac/ac-quiz/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+.node_modules
```
Comment (lfaucon) [inline ac/ac-quiz/.gitignore]: should add `dist`

File: ac/ac-quiz/src/index.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+          }
+        }
+      }
+    }
+  }
+}
+
+export const ActivityRunner = ({ config, logger, onCompletion }) => {
+  const propdef = config.MCQ.reduce(
+    (acc, x, i) => ({...acc, [i + '']: {
```
Comment (lfaucon) [inline ac/ac-quiz/src/index.js]: A bit difficult to understand if you use `x`, `i`, `y`, and `j` to name your variables. Couldn't it be `question`, `questionIndex`, `answer` and `answerIndex`?

**Pull request merged**